test/langtools/tools/doclint/tidy/InvalidName.out

```diff
@@ -1,4 +1,4 @@
 InvalidName.java:17: error: invalid name for anchor: "foo()"
- * <a name="foo()">invalid</a>
+ * <a id="foo()">invalid</a>
       ^
```

Comment:
This seems incorrect. In HTML5 all names are valid unless they contain whitespace.

Reply:
Understood.  Need to file it as another DocLint bug.

Reply:
Filed as [JDK-8256312](https://bugs.openjdk.java.net/browse/JDK-8256312)